packages/components/file-upload/multiple-file-upload.component.ts

```diff
@@ -151,16 +155,21 @@ export class KbqMultipleFileUploadComponent
         return this.hint.length > 0;
     }
 
-    constructor(
-        private cdr: ChangeDetectorRef,
-        private renderer: Renderer2,
-        @Optional() @Inject(KBQ_FILE_UPLOAD_CONFIGURATION) public readonly configuration: KbqInputFileMultipleLabel,
-        @Optional() @Inject(KBQ_LOCALE_SERVICE) private localeService?: KbqLocaleService,
-        @Optional() @Self() public ngControl?: NgControl
-    ) {
+    private cdr = inject(ChangeDetectorRef);
+    private renderer = inject(Renderer2);
+    private destroyRef = inject(DestroyRef);
+    readonly configuration: KbqInputFileMultipleLabel | null = inject<KbqInputFileMultipleLabel>(
```

Comment:
кажется что это необязательно писать, ts и так подставит нужный тип, а читается легче
```suggestion
    readonly configuration = inject<KbqInputFileMultipleLabel>(
```